jupyterlab_caip_optimizer/setup.py

```diff
@@ -0,0 +1,183 @@
+from __future__ import print_function
```

Comment:
I'd scrap this and base it on the `setup.py` files from any of the existing extension folders. This setup borrowed from core JupyterLab and was overly complex to suit what we needed to do with it.